src/boosting/gbdt.h

```diff
@@ -180,6 +199,10 @@ class GBDT: public Boosting {
   data_size_t bag_data_cnt_;
   /*! \brief Number of traning data */
   data_size_t num_data_;
+  /*! \brief Number of classes */
+  int num_class_;
+  /*! \brief Current class */
+  int curr_class_;
```

Comment:
really use this?  Or add "curr_class" as parameter in these functions who use "curr_class"?